lib/builtin.t

```diff
@@ -51,6 +51,17 @@ package FetchStoreCounter {
     is(prototype(\&builtin::is_bool), '$', 'is_bool prototype');
 }
 
+# float contants
+{
+    use builtin qw( inf nan );
+
+    ok(inf, 'inf is true');
+    ok(inf > 1E10, 'inf is bigger than 1E10');
+    ok(inf == inf, 'inf is equal to inf');
```

Comment:
[Infinity should be larger than all other values except itself and NaN](https://www.gnu.org/software/libc/manual/html_node/Infinity-and-NaN.html).